src/main/java/frc/team2412/robot/util/InterpolatingTreeMap.java

```diff
@@ -23,6 +34,60 @@ public InterpolatingTreeMap(ShooterDataDistancePoint[] dataPoints) {
         distanceBiasEntry = ShooterConstants.tab.add("Distance Bias", 0.0).withPosition(0, 0).withSize(1, 1).getEntry();
     }
 
+    /**
+     * Creates a {@link InterpolatingTreeMap} from a path to a CSV file.
+     *
+     * @param fileName
+     *            The path to the CSV file.
+     * @return An {@link InterpolatingTreeMap} from the data in the CSV file.
+     */
+    public static InterpolatingTreeMap fromCsv(String fileName) {
+        try (BufferedReader reader = Files.newBufferedReader(Paths.get(fileName))) {
+            String line;
+            InterpolatingTreeMap map = new InterpolatingTreeMap();
+
+            while ((line = reader.readLine()) != null) {
+                String[] items = line.split(",");
+                if (items.length < 3) {
+                    System.out.println("Line " + Arrays.toString(items) + " has less than 3 items, skipping...");
+                    continue;
+                } else if (items.length > 3) {
+                    System.out.println(
+                            "Line " + Arrays.toString(items) + " has more than 3 items, discarding extra data...");
+                }
+                ShooterDataDistancePoint point = new ShooterDataDistancePoint(Double.parseDouble(items[0]),
```

Comment:
It would be good to catch bad field data ie points that are not numbers (for instance if someone adds field names).